quickwit/quickwit-indexing/src/actors/indexing_service.rs

```diff
@@ -123,6 +127,7 @@ impl IndexingService {
             pipeline_handles: Default::default(),
             state: Default::default(),
             enable_ingest_api,
+            shared_resource_manager: IndexingSharedResourceManager::new(),
```

Comment:
ok 👍 . I started with indexing_directory, and split_store like here but realized we will do that for the merge pipeline too.  I was feeling like it's going to get in the way of the main actor's logic so to me it would be simpler isolating it..